source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -244,6 +253,40 @@ json TransformerInstance::cluster_metadata_callback(
   return "";
 }
 
+json TransformerInstance::base64_encode_callback(const inja::Arguments &args) const {
+  const std::string &input = args.at(0)->get_ref<const std::string &>();
+  return Base64::encode(input.c_str(), input.length());
+}
+
+json TransformerInstance::base64_decode_callback(const inja::Arguments &args) const {
+  const std::string &input = args.at(0)->get_ref<const std::string &>();
+  return Base64::decode(input);
```

Comment:
do you know if the use case requires / we may need to expose the `remove_linebreaks` option here? https://renenyffenegger.ch/notes/development/Base64/Encoding-and-decoding-base-64-with-cpp/

Reply:
I think it probably does not (seems like they need to decode an auth token from some external service?), but it's trivial to expose an optional second parameter to the user so I'm going to default `remove_linebreaks` to false and give the user the option to enable it

Reply:
Ah, so the base64 utilities that we're using here (defined in upstream Envoy) only support a single argument to `decode`, see here: https://github.com/envoyproxy/envoy/blob/main/source/common/common/base64.cc#L146

I think it's pretty reasonable to assume that the string being decoded does not contain linebreaks -- what do you think?

Reply:
I think it's fine to leave as is until a request for it comes in.. if envoy hasn't needed it yet then i doubt we will

Reply:
Sounds good. Resolving this thread for now